webviz_subsurface/datainput/_summary.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+""" time-series data-input functions
+    - Process ensemble-data (read, filter, compare)
+    - Treated as ensembleset-object (independent of n-ensembles)
+"""
+
+
+try:
+    from fmu.ensemble import EnsembleSet
+except ImportError:  # fmu.ensemble is an optional dependency, e.g.
+    pass # for a portable webviz instance, it is never used.
+
+
+import pandas as pd
+from webviz_config.common_cache import cache
+from webviz_config.webviz_store import webvizstore
+from ..datainput import scratch_ensemble
+
+
+@cache.memoize(timeout=cache.TIMEOUT)
+def load_ensemble_set(
+        ensemble_paths: tuple,
+        ensemble_set_name: str = 'EnsembleSet'):
+
+    return EnsembleSet(
+        ensemble_set_name,
+        [scratch_ensemble(ens_name, ens_path)
+         for ens_name, ens_path in ensemble_paths]
+    )
+
+
+@cache.memoize(timeout=cache.TIMEOUT)
+@webvizstore
+def get_time_series_statistics(
+        ensemble_paths: tuple,
+        time_index: str,
+        column_keys: tuple
+    ) -> pd.DataFrame:
+
+    column_keys = list(column_keys) if isinstance(
+        column_keys, (list, tuple)) else None
+
+    # Note: to be replaced by ensset.get_smry_stats()
+    # when function is available
```

Comment:
Is there an issue in `fmu-ensemble` that we can link to in this comment?

Reply:
https://github.com/equinor/fmu-ensemble/pull/19

Reply:
Good :) This link can then be added to the comment in the code.